### DISCUSSION TITLE
Use Powershell cmdlet DisplayConfig to manage display setup

### DISCUSSION THREAD

**Discussion by yufanyufan** [LizardByte/community#578] [Sunshine]
### Select Topic Area

Show & Tell

### Body

The current recommendation of managing display setup uses long list of tools, like qres, MultiMonitorTool, hdr-switch.
Some of those tools are not updated in years.

I suggest to update the doc to use DisplayConfig as example.
DisplayConfig can handle monitor switch, resolution, refresh rate, hdr, scaling. It is everything in one open source package.
https://www.powershellgallery.com/packages/DisplayConfig

Example:
powershell -c 'Set-DisplayPrimary 1'
powershell -c "Set-DisplayResolution 1 -Width $Env:SUNSHINE_CLIENT_WIDTH -Height $Env:SUNSHINE_CLIENT_HEIGHT"
powershell -c 'Set-DisplayScale 1 -Scale 200'
powershell -c 'if ($Env:SUNSHINE_CLIENT_HDR -eq \\"true\\") {Set-DisplayHdr 1 -EnableHDR}'

**Comment by ReenigneArcher** [comment]
> The current recommendation

There is NO recommendation at all. These are just examples. If you'd like you could submit this to https://github.com/LizardByte/awesome-sunshine

[a comment by an automated account is not shown]
